code/modules/surgery/organs/eyes.dm

```diff
@@ -186,37 +186,70 @@
 
 	damaged = TRUE
 
+#define NIGHTVISION_LIGHT_OFF 0
+#define NIGHTVISION_LIGHT_LOW 1
+#define NIGHTVISION_LIGHT_MID 2
+#define NIGHTVISION_LIGHT_HIG 3
+
 /obj/item/organ/internal/eyes/night_vision
-	see_in_dark = NIGHTVISION_FOV_RANGE
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	// These lists are used as the color cutoff for the eye
+	// They need to be filled out for subtypes
+	var/list/low_light_cutoff
+	var/list/medium_light_cutoff
+	var/list/high_light_cutoff
+	var/light_level = NIGHTVISION_LIGHT_OFF
 	actions_types = list(/datum/action/item_action/organ_action/use)
-	var/night_vision = TRUE
+
+/obj/item/organ/internal/eyes/night_vision/Initialize(mapload)
+	. = ..()
+	if (type != /obj/item/organ/internal/eyes/night_vision && PERFORM_ALL_TESTS(focus_only/nightvision_color_cutoffs))
+		if(length(low_light_cutoff) != 3 || \
+			length(medium_light_cutoff) != 3 || \
+			length(high_light_cutoff) != 3)
```

Comment:
these all don't fit on the same line? looks like it should be okay... maybe it looks better in the actual environment but multilining this here makes it hard to read imo for not-that-much payoff

Reply:
idk I thought it looked better, will give it another look